roles/aodh/tasks/monitoring.yml

```diff
@@ -7,10 +7,6 @@
   sensu_process_check: service=aodh-evaluator
   notify: restart sensu-client
 
-- name: aodh-expirer process check
-  sensu_process_check: service=aodh-expirer
-  notify: restart sensu-client
-
```

Comment:
Instead of just removing the code here, we need to turn this into a `state=absent` task, so that the existing check is removed.